Source/Server/MqttService.cpp

```diff
@@ -165,12 +137,12 @@ namespace MQTT {
 			return nullptr;
 		}
 
-
-		void MqttService::OnClientDisconnect(const Client& client, const Protocol::DisconnectPackage& package)
+		std::string MqttService::GenerateUniqueId()
```

Comment:
Skal fjernes da den ikke længere bruges

Reply:
Resolved by deleting method. Not used anymore